docs/sweater-comb/index.md

```diff
@@ -1,6 +1,15 @@
 # About this project
 
-COMING SOON:
+## Removing support for old endpoint versions
+
+Removing support for an old API version requires:
+1. Temporarily ignoring the endpoint version code in `.vervet.yaml`
+2. Removing the endpoint versions from `catalog-info.yaml`
+3. Removing the old OpenAPI specs.
+
+[Example PR](https://github.com/snyk/registry/pull/33489/files)
```

Comment:
As this is a public repo, it is not good practice to refer to names/files of internal services. I would prefer we included any relevant code examples in the docs themselves.